src/lib.rs

```diff
@@ -220,10 +234,18 @@ impl SandboxFS {
     /// numbers we have previously told it about.
     fn find_node(&mut self, inode: u64) -> Arc<nodes::Node> {
         let nodes = self.nodes.lock().unwrap();
-        match nodes.get(&inode) {
-            Some(node) => node.clone(),
-            None => panic!("Kernel requested unknown inode {}", inode),
-        }
+        nodes.get(&inode).expect("Kernel requested unknown inode").clone()
```

Comment:
`expect()` crashes the process. I know this is a "should not happen", but I would prefer to log and return an error here. The caller can EIO the kernel request.

(also below)

Reply:
I still don't understand why you'd like to treat what seem to be logic errors gracefully. Yes, we could hit this condition due to a bug, but we could also hit it due to bad expectations on our side.

Either way, I think crashing is the best thing to do. What I'm concerned about is that, if we ever encounter this, we are in a situation where our view of inodes (or handles) doesn't match that of the kernel. Which implies two things: we might not find the requested inodes... but we may also be handing the kernel information for the wrong inode. The latter seems pretty bad: users could peek into files they are not allowed to access if that were the case so I think it makes sense to crash as soon as possible.